help/en/html/doc/Technical/IntelliJ.shtml

```diff
@@ -719,26 +719,26 @@
       <p>To make a PR from within IntelliJ IDEA, select "Create
       Pull Request" from the "VCS" -&gt; "Git" menu (it's the one but
       last item, supported by the Git plugin).<br>
-      Note: Be sure to select your own repository if you see multiple options
-      to choose from in this submenu:<br>
-      <img src="images/IntelliJ/IjPullViaOwnRepo.png" width="622" height="52" alt="IntelliJ Git Menu"></p>
+      Note: Be sure to select your own repository <code>origon</code>if you see
```

Comment:
No big deal and I'm happy to approve this PR despite this, but shouldn't origon be origin?

Reply:
Yes, well spotted, thanks. Now fixed